daskernetes/__init__.py

```diff
@@ -1,26 +1,30 @@
 import logging
 import tornado
+import socket
 import argparse
 from urllib.parse import urlparse
 
-
 from distributed import Client
-from distributed.deploy import Adaptive
+from distributed.deploy import Adaptive, LocalCluster
 from distributed.utils import LoopRunner, sync
 from distributed.scheduler import Scheduler
 from kubernetes import client, config
 
+logger = logging.getLogger(__name__)
 
 class KubeCluster(object):
     def __init__(
             self,
-            name,
-            namespace,
-            worker_image,
-            worker_labels,
-            scheduler_address
+            name='dask',
+            namespace='dask',
+            worker_image='daskdev/dask:latest',
+            worker_labels=None,
+            n_workers=0,
+            threads_per_worker=1,
+            host='0.0.0.0',
     ):
-        self.log = logging.getLogger("distributed.deploy.adaptive")
+        self.cluster = LocalCluster(ip=host or socket.gethostname(),
+                                    n_workers=0)
```

Comment:
LocalCluster also sets up things for us like the dashboard, which can be pleasant.  I'm happy to do things more raw with `Scheduler` though if preferred.